label_studio/io_storages/s3/models.py

```diff
@@ -17,9 +17,8 @@
 from io_storages.s3.utils import get_client_and_resource, resolve_s3_url
 from tasks.validation import ValidationError as TaskValidationError
 from io_storages.serializers import StorageAnnotationSerializer
-from tasks.serializers import TaskSerializerBulk
 from tasks.models import Annotation
-from data_import.serializers import ImportApiSerializer
+from data_export.serializers import ExportDataSerializer
```

Comment:
We need to combine ExportDataSeriaizer with tasks.serlializers.TaskSerializer because of the duplicated code part
```
        if settings.DATA_UNDEFINED_NAME in data and data_types_keys:
            key = list(data_types_keys)[0]
            data[key] = data[settings.DATA_UNDEFINED_NAME]
            del data[settings.DATA_UNDEFINED_NAME]
```

Also we have to add 
```
    file_upload = serializers.ReadOnlyField(source='file_upload_name')
```
to TaskSerializer.